tests/unit/controllers/start/expertise-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:start/expertise', 'Unit | Controller | start/expertise', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+// Replace this with your real tests.
+test('it exists', function(assert) {
+  let controller = this.subject();
+  assert.ok(controller);
+});
```

Comment:
Generally,  if It's a unit test, then I would only test the code that doesn't involve other parts of the app. Otherwise, if there is some behavior that needs testing, but it interacts with other parts, then I'd make an integration test. In this case, we'd likely want the integration test that ensures we have a user property.

In the case of the interests controller, there are two actions that need testing. Since they deal with the store, again, I'd go with an integration test.